robber/matchers/__init__.py

```diff
@@ -5,12 +5,12 @@
     'called_once',
     'called_once_with',
     'called_with',
-    'contain'
+    'contain',
     'equal',
-    'ever_called'
+    'ever_called',
```

Comment:
`assert_called(*args, **kwargs)`
Assert that the mock was called at least once.

`assert_any_call(*args, **kwargs)`
assert the mock has been called with the specified arguments.

The assert passes if the mock has ever been called, unlike `assert_called_with()` and `assert_called_once_with()` that only pass if the call is the most recent one, and in the case of `assert_called_once_with()` it must also be the only call.

Source: https://docs.python.org/3/library/unittest.mock.html